### PR TITLE
Add Github Workflow and fix mcprotocollib dependency

### PR DESCRIPTION
**Pull request by louboecker**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: gradle-ci
+on: [ push ]
```
Comment (Cubxity) [inline .github/workflows/build.yml]: This should only run on main/master

File: build.gradle.kts [hunk 2]
```diff
@@ -8,14 +8,15 @@ version = "0.0.1"
 
 repositories {
     mavenCentral()
-    maven("https://jitpack.io")
+    maven("https://jitpack.io/")
```
Comment (Cubxity) [inline build.gradle.kts:11]: Remove jitpack

**Review by Cubxity — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 2 above)
Comment (DRSchlaubi) [reply build.gradle.kts:11]: MCProtocolLib has a dependency requiring jitpack, even though itself not being on jitpack iirc

Comment (Cubxity) [reply build.gradle.kts:11]: That's reasonable then

**Review by Cubxity — APPROVED** [review]

**Pull request merged**